packages/checkbox/src/CheckboxMixin.ts

```diff
@@ -78,6 +83,19 @@ export function CheckboxMixin<T extends Constructor<ReactiveElement>>(
                 />
             `;
         }
+
+        protected override updated(changes: PropertyValues): void {
+            super.updated(changes);
+            if (
+                changes.has('readonly') &&
+                (typeof changes.get('readonly') !== 'undefined' ||
+                    this.readonly)
+            ) {
+                if (this.readonly) {
+                    this.inputElement.disabled = this.readonly;
+                }
```

Comment:
This doesn't look like it supports removing `readonly`?

Reply:
Yup. great catch! update to allow support for both setting and removing readonly